crates/types/src/services/p2p.rs

```diff
@@ -52,10 +52,10 @@ pub enum GossipsubMessageAcceptance {
     /// Report whether the gossiped message is valid and safe to rebroadcast
```

Comment:
"Report whether" is weird wording here. Doesn't it just "Accept" the message?

Reply:
Agreed. I wonder if we should even use verbs in these comments like "Report". This is a data structure, and not a behavior we're documenting.

I've reworked the comments a bit now. What do you think of the following?
```rust
/// Status of a message received via Gossip
#[derive(Clone, Copy, PartialEq, Eq, Debug, Hash)]
pub enum GossipsubMessageAcceptance {
    /// The gossiped message is valid and safe to rebroadcast.
    Accept,
    /// The gossiped message is invalid and should be ignored.
    Ignore,
    /// The gossiped message is invalid and anyone relaying it should be penalized.
    Reject,
}
```

Reply:
Changed in https://github.com/FuelLabs/fuel-core/compare/6c589b8ed39d4cb4e628b08a7c72a2543df61102..7cf9472ca0fa01793a9c025850c778dbd7bf15ff